patches/minecraft/net/minecraft/world/server/ServerWorld.java.patch

```diff
@@ -61,6 +61,15 @@
           }
  
           this.func_229856_ab_();
+@@ -338,7 +343,7 @@
+       this.func_147488_Z();
+       this.field_211159_Q = false;
+       iprofiler.func_219895_b("entities");
+-      boolean flag3 = !this.field_217491_A.isEmpty() || !this.func_217469_z().isEmpty();
++      boolean flag3 = !this.field_217491_A.isEmpty() || net.minecraftforge.common.world.ForgeChunkManager.hasForcedChunks(this);
```

Comment:
Why does this not matter anymore: `!this.func_217469_z().isEmpty()`, I more orless expected this PR to just extend vanilla behaviour, and append another or case. Instead it also removes one here.

Not saying this is wrong, but a bit of documentation on why this happens here is needed I think, since it feels counter intuitive.